unix/stddialogs.c

```diff
@@ -56,6 +56,11 @@ static void msgbox(GtkWindow *parent, const char *title, const char *description
 		type, buttons,
 		"%s", title);
 	gtk_message_dialog_format_secondary_text(GTK_MESSAGE_DIALOG(md), "%s", description);
+	if (type == GTK_MESSAGE_ERROR) {
+		gtk_message_dialog_set_image(GTK_MESSAGE_DIALOG(md),
+			gtk_image_new_from_icon_name("dialog-warning", GTK_ICON_SIZE_DIALOG));
```

Comment:
```suggestion
			gtk_image_new_from_icon_name("dialog-error", GTK_ICON_SIZE_DIALOG));
```
Should this not be `dialog-error`?
Not sure what older GTK versions used to display, but I would expect an error symbol for uiMsgBoxError.

Reply:
> Looks good apart from the warning/error image. I presume the code would simply override the stock image on older Gtk3 versions?

Yes will just override in that case.

> Should this not be dialog-error?
> Not sure what older GTK versions used to display, but I would expect an error symbol for uiMsgBoxError.

Intentionally used `dialog-warning` instead of `dialog-error` here, as the Gtk stock icon for `dialog-error` maps to a somewhat misleading "stop sign" symbol instead of an "error" symbol:

<img src="https://user-images.githubusercontent.com/11763424/236595213-c9b63509-70e9-406a-a9db-04e9fcbdaa0b.png" height="151" width="552">

`dialog-warning` matches the other platforms best as well:

<img width="262" alt="darwin_errmsg" src="https://user-images.githubusercontent.com/11763424/236595356-ff431f11-c8f6-453e-992a-1fc273dc1b52.png">

Reply:
> `dialog-warning` matches the other platforms best as well:

Interesting that that is the way `NSCriticalAlert` is displayed on darwin. Maybe not to scare people?

This is what the dialog looks like on Windows 7, not sure about newer platforms:
![screenshot](https://user-images.githubusercontent.com/286606/236643651-bc63153a-8907-43ef-bf74-23eb1124de87.png)

I guess having both a warning and  an error box would make sense?

Reply:
> not sure about newer platforms

Here it is on Windows 11, nearly identical:

<img width="357" alt="windows_msgboxerr" src="https://github.com/libui-ng/libui-ng/assets/11763424/580103ce-3270-46a2-b6e8-bafad7b66230">


> I guess having both a warning and an error box would make sense?

Yes, `uiMsgBoxWarning()` would be a good addition to the API as well.